main.go

```diff
@@ -53,11 +102,28 @@ func getGroupGifs(c *echo.Context) error {
 }
 
 func createGroup(c *echo.Context) error {
-    res := ResponseTemplate{}
-    group := Group{}
+    g := Group{groupSeq, c.Form("name")}
+
```

Comment:
1. Created an Amazon S3 bucket where I can upload the gifs, and return a URL to save within Redis.
2. Curious still about this question.